config/models.go

```diff
@@ -44,6 +47,7 @@ type AppConfig struct {
 	EnableAdvancedSetup      bool   `envconfig:"ENABLE_ADVANCED_SETUP" default:"true"`
 	AutoUnlockPassword       string `envconfig:"AUTO_UNLOCK_PASSWORD"`
 	LogDBQueries             bool   `envconfig:"LOG_DB_QUERIES" default:"false"`
+	BoltzApi                 string `envconfig:"BOLTZ_API" default:"https://api.boltz.exchange"`
```

Comment:
Should we address the NETWORK change mentioned in the PR?

We could remove the `LDK_NETWORK` default but still keep that config, and also add `NETWORK` (also without a default).

Then when initializing the config, check if either are set and use that, otherwise set the value to "bitcoin".

Reply:
Done!